src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptionsExtensions.cs

```diff
@@ -99,10 +99,12 @@ public static OtlpExporterTransmissionHandler<TraceOtlpCollector.ExportTraceServ
             ? httpTraceExportClient.HttpClient.Timeout.TotalMilliseconds
             : options.TimeoutMilliseconds;
 
-        return new OtlpExporterTransmissionHandler<TraceOtlpCollector.ExportTraceServiceRequest>(exportClient, timeoutMilliseconds);
+        return enableRetry
```

Comment:
Given the repetition of enableRetry and timeoutMilliseconds across three methods, is it feasible to consolidate them into a common method?

Reply:
No, They differ by the return type.